<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;Century Schoolbook L&quot;;
background-color: qlineargradient(spread:pad, x1:0.476, y1:0.704682, x2:0.481, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="Play_fixedframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>751</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background : transparent;
border:none;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Playing_label">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>40</y>
       <width>301</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color : black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Switch_View">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>42</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Switch_View{
background-color:transparent;
background-image: url(:/img/switchview.png);
border:none;

}
#Switch_View:checked{

            background-image: url(:/img/switchviewoff.png);

}
#Switch_View:focus {
            border: none;
            outline: none;
    }
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Next">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>42</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : none;
background-image: url(:/img/nextone.png);
outline : none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Play_toggle">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>42</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Play_toggle{
background-color:transparent;
border:none;
	background-image: url(:/img/play.png);
}
#Play_toggle:checked{
	background-image: url(:/img/pause.png);

}
    #Play_toggle:focus {
            border: none;
            outline: none;
    }
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Previous">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>42</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/img/previousone.png);
border : none;
outline : none;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>Next</zorder>
    <zorder>Play_toggle</zorder>
    <zorder>Previous</zorder>
    <zorder>Switch_View</zorder>
    <zorder>Playing_label</zorder>
   </widget>
   <widget class="QFrame" name="Volume_fixedframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background :transparent;
border:none;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>241</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Volume_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color : white;
background :transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Detailed_Viewframe">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>711</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background :transparent;
border:none;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="Seek_bar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>631</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="Favourite">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    #Favourite:focus {
            border: none;
            outline: none;
    }



</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Repeat">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
    #Repeat:focus {
            border: none;
            outline: none;
    }

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Shuffle">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>140</y>
       <width>55</width>
       <height>55</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
   #Shuffle:focus {
            border: none;
            outline: none;
    }
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Meta_Data">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Start_Time">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>271</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color : white;
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="numofsongs">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>270</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color : white;
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="End_Time">
     <property name="geometry">
      <rect>
       <x>615</x>
       <y>270</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color : white;
background : transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>Start_Time</zorder>
    <zorder>numofsongs</zorder>
    <zorder>End_Time</zorder>
    <zorder>Seek_bar</zorder>
    <zorder>Favourite</zorder>
    <zorder>Repeat</zorder>
    <zorder>Shuffle</zorder>
    <zorder>Meta_Data</zorder>
   </widget>
   <widget class="QFrame" name="HomeView_Frame">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>711</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background :transparent;
border:none;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="AllSongs">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#AllSongs{
color:white;
}</string>
     </property>
     <property name="text">
      <string>All Songs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Albums">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Albums{
color:white;
focus {
            border: none;
            outline: none;
}
}</string>
     </property>
     <property name="text">
      <string>Albums</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Artists">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>10</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Artists{
color:white;
}</string>
     </property>
     <property name="text">
      <string>Artists</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Favourites">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>97</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Favourites</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Album_Artist_Widget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>641</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#Album_Artist_Widget::item{


color:white;
background :transparent;
}</string>
     </property>
    </widget>
    <widget class="QListWidget" name="Songs_Widget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>641</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
#Songs_Widget::item{

color:white;
background :transparent;
}
#Songs_Widget::item::selected{

color:orange;

}</string>
     </property>
    </widget>
   </widget>
   <zorder>Play_fixedframe</zorder>
   <zorder>Volume_fixedframe</zorder>
   <zorder>HomeView_Frame</zorder>
   <zorder>Detailed_Viewframe</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
